### DISCUSSION TITLE
Variants - Redirect issues: Gitbook Docs

### DISCUSSION THREAD

**Discussion by Ifycode** [GitbookIO/community#375] [0. General bugs and questions]
Hi, I'm having an issue with variants based on what is written in the gitbook docs: https://docs.gitbook.com/publishing/share/collection-publishing

#

The forest admin example was used in the gitbook docs. 
- Click on **_v7 (v6 Rails)_** from the variant dropdown, and then go to the premium features page: https://docs.forestadmin.com/documentation/v/v7-v6-rails/reference-guide/premium-features
- While still on the v7's premium features page, try to access another variant through the dropdown. This page does not exist in other variants, so it redirects to a page that exists in the variants. Therefore, the other variants are able to open up successfully and without errors (i.e. it goes to the how it works page of those variants since they don't have premium features page).

#

In my own case, it doesn't redirect. It just tries to look for the same page and url for the variant page doc I'm coming from in the variant doc I'm going to. And because a page does not exist, it brings up the `page not found` error page. I have to always click on the variant a second time after the error page occurs to get an existing variant page to successfully show up.

I've looked through the gitbook docs but can't seem to locate anything about how to go about such redirects for variants. How were they able to achieve that? Can you help direct me?

#

Reference Discussion: https://github.com/orgs/GitbookIO/discussions/350

**Comment by aikoii** [comment]
👋 @Ifycode, answered here https://github.com/orgs/GitbookIO/discussions/350#discussioncomment-7358577

**Reply by Ifycode** [reply]
Seen. Thank you.

**Comment by aikoii** [comment]
Thanks!  I'll be closing this in favor of the original post 🙏

**Reply by Ifycode** [reply]
Thank you @aikoii No problem.
I've added more responses and my issues and concerns to the original post.
